### DISCUSSION TITLE
How to translate boolean config variables to strings?

### DISCUSSION THREAD

**Discussion by sschuberth** [batect/batect#1357] [Q&A] [answered]
As a follow-up to [this comment](https://github.com/batect/batect/pull/990#issuecomment-1259055385) of mine, is there currently an elegant way to map a boolean config variable to strings? More concretely, to something like "--option" if `true` and "" (empty string) if `false`, so these strings can be passed as arguments to commands that I run?

**Comment by charleskorn** [comment, marked as answer]
At the moment, this isn't supported. I'd suggest moving any logic like that into something like a shell script.
